.github/workflows/unlabel.yml

```diff
@@ -0,0 +1,11 @@
+name: Remove Label
+on:
+  issue_comment:
```

Comment:
I would have done it on PR commits, but perhaps makes more sense to remove it on comments. Is it possible to avoid running it when one of us is commenting?

Reply:
doing it on PR commits is fairly annoying to do as it will add that one action in the checks for the PR and for example if our CI is broken and no output is result has been sent yet or github broke somehow it will show the PR as "checked and good to go" even though it is very not. There is a github feature request asking for a way to hide them but it hasn't seen any activity in 2 years: https://github.com/orgs/community/discussions/26246

> Is it possible to avoid running it when one of us is commenting?

I've done that though. I've used github variables (https://docs.github.com/en/actions/learn-github-actions/variables) to store that information and the name of the label too, to make it easier to modify it in the future.